src/pages/stacks-blockchain/transactions.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Transaction generation
+description: Guide to Stacks 2.0 transactions
+---
+
+## Introduction
+
+Transactions are the fundamental unit of execution in the Stacks blockchain. Each transaction is originated from a standard account, and is retained in the Stacks blockchain history for eternity. This guide helps you understand Stacks 2.0 transactions.
```

Comment:
[nit] readers won't know what we mean by "standard account". Is the intention to link this back to the `accounts` page once that's populated?

Reply:
Yes, I updated that as well